bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/database/ShadeCapabilitiesDatabase.java

```diff
@@ -155,7 +155,7 @@ public int getCapabilities() {
          * @return 'typeCapabilities'.
          */
         public int getCapabilitiesOverride() {
-            return capabilitiesOverride;
+            return capabilitiesOverride < 0 ? capabilities : capabilitiesOverride;
```

Comment:
My understanding is that this method should simply provide the raw override value - the same way as `getCapabilities()` provides the raw capabilities (previous method). The fallback/override logic is in calling method, and probably should be kept and extended there?
https://github.com/openhab/openhab-addons/blob/0a2a85b448e9804fb88fc51e8f8ff22859cafe60/bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/database/ShadeCapabilitiesDatabase.java#L335-L352

Also it would be nice if this is covered by a test.